templates/index.html

```diff
@@ -0,0 +1,59 @@
+{% extends "layout.html" %}
+{% block javascript %}
+<script type="application/javascript" language="javascript">
+document.addEventListener('DOMContentLoaded', (event) => {
```

Comment:
This looks good, Do you mind if I refactor this, in a following PR, so we have one calls to an external js file, instead of doing this inline?